includes/widgets/icon.php

```diff
@@ -331,6 +352,27 @@ protected function register_controls() {
 			]
 		);
 
+		$this->add_control(
+			'icon_fit_to_size',
```

Comment:
One of these two new controls needs to have its name changed, they have the same label.
@avivu Can you consult please?